src/App.tsx

```diff
@@ -1,22 +1,18 @@
-import * as React from 'react';
-import './App.css';
-
-import logo from './logo.svg';
+import * as React from "react";
+import "./App.css";
+import intro from "./images/intro.png";
 
 class App extends React.Component {
-  public render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.tsx</code> and save to reload.
-        </p>
-      </div>
-    );
-  }
+    public render() {
```

Comment:
I see that this is 4-spaced. I would prefer it if we kept it to 2 spaces as that is general JS standards. I will add an `.editorconfig` file to enforce this in the future.

Reply:
Let me know if you're good to change it now, otherwise we can do it in the future @bhuwanoza

Reply:
ok i'll change it now

Reply:
i will format it later..

Reply:
Sounds good. I will approve it and merge it in!